src/tye/Program.DeployCommand.cs

```diff
@@ -104,7 +113,11 @@ private static async Task DeployApplicationManifestAsync(OutputContext output, A
             }
 
             output.WriteDebugLine("Running 'kubectl apply'.");
-            output.WriteCommandLine("kubectl", $"apply -f \"{tempFile.FilePath}\"");
+            var cmd = $"apply -f \"{tempFile.FilePath}\"";
+            if (!String.IsNullOrWhiteSpace(ns)) {
+                cmd = cmd + $" --namespace {ns}";
```

Comment:
I don't think you should include the namespace in the output here, as the namespace information is in the tempFile here. Maybe changing the above log for `Running 'kubectl apply'.` to `Running 'kubectl apply' on namespace {ns}.

Reply:
yeah, it's little surprising to me that this ends up in the manifest and the command line. Is there a reason for that?

We actually have a few code paths that interact with manifest generation - have a currently undocumented command that generates the manifest to a file on disk https://github.com/dotnet/tye/blob/master/src/tye/Program.GenerateCommand.cs

so it seems better to thread this through `GenerateKubernetesManifestStep` than the command line

Reply:
done.